UtFoM_Project2/Assets/Scripts/PlayerMovement.cs

```diff
@@ -55,45 +49,10 @@ public void Knock(float kbTime, float damage)
 		}
 	}
 
-	private IEnumerator KnifeAttackCo()
-	{
-		animator.SetBool("knifing", true);
-		currentState = PlayerState.knife;
-		yield return null;
-		animator.SetBool("knifing", false);
-		yield return new WaitForSeconds(.08f);
-		currentState = PlayerState.idle;
-	}
-
-	private IEnumerator HammerAttackCo()
-	{
-		animator.SetBool("hammering", true);
-		currentState = PlayerState.hammer1;
-		yield return null;
-		animator.SetBool("hammering", false);
-		yield return new WaitForSeconds(0.83f);
-		currentState = PlayerState.idle;
-	}
-
-	void UpdateAnimationAndMove()
-	{
-		if (change != Vector3.zero)
-		{
-			MoveCharacter();
-			animator.SetFloat("moveX", change.x);
-			animator.SetFloat("moveY", change.y);
-			animator.SetBool("moving", true);
-		}
-		else
-		{
-			animator.SetBool("moving", false);
-		}
-	}
-
     // Start is called before the first frame update
     void Start()
     {
-		currentState = PlayerState.idle;
+		//currentState = PlayerState.idle;

```

Comment:
Do we want to leave this comment in, or can we delete it?

Reply:
We can remove it because we are not currently using an in-code state machine.